yarn.lock

```diff
@@ -7795,7 +7795,7 @@ file-loader@3.0.1, file-loader@^3.0.1:
     loader-utils "^1.0.2"
     schema-utils "^1.0.0"
 
-file-saver@eligrey/FileSaver.js#1.3.8:
+"file-saver@github:eligrey/FileSaver.js#1.3.8":
```

Comment:
Why this diff again? 😭

Reply:
Jesus Christ. I have no idea.

Reply:
Why does this PR contain yarn.lock changes at all? Did you do `yarn upgrade`? I'd suggest not to include package upgrades in regular PRs to avoid all the merge issues when 4 different people change yarn.lock in their PRs. Instead, create a separate PR that only does upgrades and nothing else.

Reply:
The thing is that line appeared if I deleted the yarn.lock and did `yarn`, I never performed a `yarn upgrade`. In all my PRs I did the following:

1. Delete my `yarn.lock`, so that minor versions upgrade. I believe @apanizo also does this.

Reply:
The thing is that line appeared if I deleted the yarn.lock and did yarn, I never performed a yarn upgrade. In all my PRs I did the following:

1. Delete my yarn.lock, so that minor versions upgrade. I believe @apanizo also does this.
2. Perform `yarn`, which I believe is the same as `yarn install`.
3. Manually edit the yarn.lock to change this faulty line.

Reply:
Deleting `yarn.lock` + running `yarn`/`yarn install` is the same thing as sunning `yarn upgrade` and there is no reason to do it in the normal workflow.

Reply:
Ok, that was where I was messing up, don't know if @apanizo is having the same issue.

Reply:
This is what I am doing, I started to do this because I had problems when I was using `yarn link` and custom local modifications via `yarn add "/../../......../my/library"`

What do you guys think, should we continue doing the upgrade thing on each PR? Or just stop and every Friday create a PR upgrading stuff? I would not like to spend more time without upgrading libs.

Reply:
I think the workflow is like this

1. All 4 people do: `git pull && yarn install && yarn build` and push changes
2. 1 person does (once a day or once a week, I don't mind): In the case you have a good reason for upgrading, use `yarn upgrade`, revert the file-saver diff, commit and push "Upgrade dependencies" – no other change in the PR.

Reply:
Sounds good to me.